### ISSUE TITLE
No title for admin course

### ISSUE THREAD

**Issue by Alex-Jordan** [openwebwork/webwork2#2497]
It seems that my admin course has no course title. (There is no `courseTitle` row in the `admin_setting` table. Nor is there a `courseInstitution` row.) This is an admin course that propagated from earlier versions of WW (although the admin upgrade script was applied each time), and I am unsure if a new admin course on a new installation would not have a title.

This is not a problem, because when you are in the admin course, nothing tries to access its title.

But now it will be common to copy the "admin" course to some new course with a different name that is destined to be the new admin course (just with a new name). When you copy the "admin" course, it would be natural to "select all" so that you copy everything possible. This is leading to the newly created course also having no title. And now if you try to enter that course, there are perl warnings about the title being undefined. Soon you will toggle the config setting to make this the new admin course, and then it won't matter again. But it's unsettling if you go to this course to check on things and get these warnings.

Once logged in to the new course, you can go to Course Config and give it a title. That is, if you understood the warnings well enough to know to try that.

You cannot do this in the original admin course while it is still the active admin course, because we don't give access to Course Config. (Well, you can still make your browser go to `webwork2/admin/instructor/config` but we don't offer a TOC link to the Config page.)

Long story short: I think that the update admin script should give the admin course a title now, if it does not already have one. This means the script would first check if the course has a title, and I don't know sql scripting well enough to do that. Also, if no title exits, what title should it get? An empty string? `Admin`? `Course Administration`?

**Comment by Alex-Jordan** [comment]
And if it is not already done, the course integrity check (like what is run when you are in the admin course and go to Upgrade Courses) should be looking to see that there is a `courseTitle` row in the `settings` table. Since without such a row, there will be this perl warning.

**Comment by somiaj** [comment]
This was fixed in #2516.
